### DISCUSSION TITLE
how to load custom config or change public directory ?

### DISCUSSION THREAD

**Discussion by Metam** [alex8088/electron-vite#244] [Q&A]
I have a vue 3 project with a custom config, it is located here: /public/.env.js
```
window['_ENV_'] = { ... }
```
I want to make a separate config file for electron

how can I import the required .env.electron.js, 
or specify a special public directory "public_electron", 
or just add new properties to the window object?

**Comment by alex8088** [comment]
`import xx from '.env.js'`

**Reply by Metam** [reply]
i need this import only for electron, without adding new code to application, i tried to use it in preload - nothing worked, public folder is added to renderer by default - can i set the path to it?
